cui-ra/index.html

```diff
@@ -44,14 +44,19 @@
         </div>
       </div>
     </header>
-
-
     <div class="govuk-width-container">
       <main class="govuk-main-wrapper" id="main-content" role="main">
         <h1 class="govuk-heading-xl">Sorry, the service is unavailable</h1>
```

Comment:
Please update the wording to - Sorry, there is a problem with the service

Reply:
Updated.